crypto/store/loader_file.c

```diff
@@ -179,6 +179,7 @@ static OSSL_STORE_INFO *try_decode_PKCS12(const char *pem_name,
     OSSL_STORE_INFO *store_info = NULL;
     STACK_OF(OSSL_STORE_INFO) *ctx = *pctx;
 
+    *matchcount = 0;
```

Comment:
Ah, that's true.  Somehow I thought the try_decode functions were visible outside this file.